### PR TITLE
Add hostname from IP address to search and filters

### PR DESCRIPTION
**Pull request by ar4s**

### CODE REVIEW

File: src/ralph/admin/filters.py [hunk 1]
```diff
@@ -535,6 +535,32 @@ def queryset(self, request, queryset):
...
+    title = _('Hostname')
+    parameter_name = 'hostname'
+    template = 'admin/filters/text_filter.html'
+
+    def queryset(self, request, queryset):
+        if not self.value():
+            return queryset
+        queries = [
+            Q(hostname__icontains=self.value()) |
+            Q(ethernet_set__ipaddress__hostname__icontains=self.value())
```
Comment (mkurek) [inline src/ralph/admin/filters.py]: could you add this condition (for hostname of ip) to DCHostHostnameFilter https://github.com/allegro/ralph/blob/ng/src/ralph/data_center/admin.py#L79 ?

**Comment by mkurek** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**